docs/modules/development/pages/docs/introduction.adoc

```diff
@@ -4,34 +4,57 @@
 Each section will produce a HTML output in the file system that is generated in the `target/generated` sources folder.
 
-The chosen file format for documentation is AsciiDoc (http://www.methods.co.nz/asciidoc[Asciidoc Homepage]).
-Document files use the `.adoc` file extension.
+We use Antora to build our documentation. 
+The file format for documentation is AsciiDoc, with the `.adoc` file extension.
+For information on syntax, see Antora's https://docs.antora.org/antora/2.3/asciidoc/asciidoc/[AsciiDoc Primer] or look at how the existing `.adoc` files work. 
 
-Note that there are different ways to contribute documentation, each suitable for the different use cases:
+Note that there are different ways to contribute documentation, each suitable for different use cases:
 
-* Tutorials and How To's should be published in our https://opennms.discourse.group/c/knowledgebase[knowledge base].
-For example:
-you want to describe how to use the Net-SNMP agent and the SNMP monitor from OpenNMS to solve a special use case with OpenNMS.
+* Publish tutorials and how to's in our https://opennms.discourse.group/c/knowledgebase[knowledge base].
+For example, you want to describe how to use the Net-SNMP agent and the OpenNMS SNMP monitor to solve a special use case with OpenNMS. 
+This is a great way to help other OpenNMS users who might have similar questions.
 
-* The documentation in the source code should be formal technical documentation.
-The writing style should be accurate and concise.
-However, ensure that you explain concepts in detail and do not make omissions.
+* Contribute formal technical documentation related to the source code on https://github.com/OpenNMS/opennms[GitHub]. 
+
+* Pick up "quickwin-docs" issues from our Jira documentation backlog.
+These are issues that should require only a short amount of time to address. 
+
+== Technical documentation tips 
+
+Technical documentation should be accurate and concise, but still explain concepts in detail. 
+
+A good way to structure your contribution is as follows: 
+
+* Descriptive heading (for example, “Create a Minion” or “SnmpCollector”) 
+* Brief description of what you are documenting. 
+If this is a new feature, include why it is important to users (for example, “This feature lets users validate and tune their business service hierarchies until they achieve the desired status propagation.”) 
+* Steps to complete the procedure, or technical content such as parameters, configuration, and examples 
+
+Minimize the use of screenshots. 
+Include them only to illustrate a concept that may be difficult to understand or something that is not easy to locate in the UI.  
 
 [[gd-docs-file-structure]]
-== File Structure in opennms-doc
+== File structure in the opennms repository 
+
+Documentation is in the `docs\modules` directory. 
+Within this directory, you will find the `/modules` directory, that contains all the technical documentation associated with the source code in this repository. 
```

Comment:
We describe here from the relative path from `docs`. The `/` in Linux/Unix is an absolute path and means the root of the whole file system and might be confusing. If you want to use a relative path to `docs` you can use `./` instead of  `/` here.

Reply:
> My Grammarly told me that -> which :) As a native you probably know better if it's a false positive :D

You are correct. :-)